doc: standardize config option names with interstitial underscores (#38580)
ceph/ceph: doc/radosgw/config-ref.rst

:Default: ``30``

``rgw_mime_types_file``

> :Description: The path and location of the MIME types file. Used for Swift

Reviewer: ```suggestion
:Description: The path and location of the MIME-types file. Used for Swift
```

Author: I'm less convinced re this one, but not enough to dissent

:Default: ``false``

``rgw_log_object_name``

> :Description: The logging format for an object name. See ma npage

Reviewer: ```suggestion
:Description: The logging format for an object name. See manpage
```

Author: Respect ma authorata

.. versionadded:: Nautilus

The ``civetweb`` frontend has a threading model that uses a thread per

> connection and hence automatically throttled by ``rgw_thread_pool_size``

Reviewer: ```suggestion
connection and hence is automatically throttled by the ``rgw_thread_pool_size``
```

Author: ack
